src/planner/binder/statement/bind_create.cpp

```diff
@@ -46,7 +46,7 @@
 
 namespace duckdb {
 
-void Binder::BindSchemaOrCatalog(ClientContext &context, string &catalog, string &schema) {
+void Binder::BindSchemaOrCatalog(string &catalog, string &schema) {
```

Comment:
I don't follow why this needs to be made non-static, is the `CatalogEntryRetriever` used here? We are creating new binders in a number of places now because of this change